README.md

```diff
@@ -44,10 +44,7 @@ let res = dynamo_client
     .index_name("my_index")
     .set_key(primary_key)
     .send()
-    .instrument(AwsSpan::build(
-        AwsTarget::Dynamo("table_name"),
-        "GetItem",
-    ))
+    .instrument(DynamoDBOperation::get_item("table_name"))
```

Comment:
I didn't like using static strings for AWS operations since we have a well defined list of all available options.

My first thought was to use an enum, but different operations require different metadata (span attributes), so I went with named constructors instead. And I decided to use macros to generate them since they are very somilar.